src/styles/StyleUtils.ts

```diff
@@ -278,28 +286,28 @@ function getZoomSizingStyle(
     containerHeight: number,
     containerWidth: number,
     isLoading: boolean,
-): ViewStyle | CSSProperties | undefined {
+): ViewStyle | undefined {
     // Hide image until finished loading to prevent showing preview with wrong dimensions
     if (isLoading || imgWidth === 0 || imgHeight === 0) {
         return undefined;
     }
-    const top = `${Math.max((containerHeight - imgHeight) / 2, 0)}px`;
-    const left = `${Math.max((containerWidth - imgWidth) / 2, 0)}px`;
+    const top = `${Math.max((containerHeight - imgHeight) / 2, 0)}px` as DimensionValue;
```

Comment:
why did we need to create `DimensionValue` in react-native types?

Is this necessary for `react-native-web` or is this something we added just because it's nice to have?

Reply:
We didn't create this type it's coming from react-native itself.
I think here its similar situation to [here](https://github.com/Expensify/App/pull/27660#discussion_r1349868629) we can't augment the type so we need to assert it